Zopa.ServiceDiagnostics/Health/IHealthCheckRunner.cs

```diff
@@ -1,9 +1,12 @@
+using System.Threading;
 using System.Threading.Tasks;
 
 namespace Zopa.ServiceDiagnostics.Health
 {
     public interface IHealthCheckRunner
     {
         Task<HealthCheckResults> DoAsync();
+
+        Task<HealthCheckResults> DoAsync(CancellationToken cancellationToken);
```

Comment:
I'd be more comfortable with one method

`Task<HealthCheckResults> DoAsync(CancellationToken cancellationToken = CancellationToken.None);`